book/controller.rst

```diff
@@ -438,7 +438,7 @@ If you want to redirect the user to another page, use the ``redirectToRoute()``
         return $this->redirectToRoute('homepage');
 
         // redirectToRoute is equivalent to using redirect() and generateUrl() together:
-        // return $this->redirect($this->generateUrl('homepage'), 301);
```

Comment:
Would you like to open a pull request for that change? By the way, I do not think we need a complete example for that, but we could add a note talking about the fact that redirects by default are responses with a 302 status code but that you can change them through the last parameter of both methods.